egs2/edacc/asr1/README.md

```diff
@@ -0,0 +1,37 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# EDACC RECIPE
+
+This is the automatic speech recognition recipe of the Edinburgh International Accents of English Corpus (EdAcc) in [EdAcc dataset](https://groups.inf.ed.ac.uk/edacc/index.html#contribute-section).
+
+Before running the recipe, please download version 1.0 from https://datashare.ed.ac.uk/handle/10283/4836 and unzip to a folder named "downloads" in this directory.
+
+# RESULTS
+## Environments
+- date: `Wed Dec 25 13:14:09 EST 2024`
+- python version: `3.10.15 (main, Oct  3 2024, 07:27:34) [GCC 11.2.0]`
+- espnet version: `espnet 202409`
+- pytorch version: `pytorch 2.0.1`
+- Git hash: `70d2c9cd76ea066fe5c84d148799b6c94e58f57c`
+  - Commit date: `Thu Dec 12 00:28:13 2024 -0500`
+
+## exp/asr_train_asr_wavlm_transformer_raw_en_bpe3884_sp
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_asr_model_valid.acc.ave/test|9300|163389|56.1|31.3|12.6|7.6|51.5|87.3|
```

Comment:
The WER seems much worse than what the paper stated. Is there a possible reason for that?

Reply:
The Reason is [here](https://github.com/espnet/espnet/pull/5996#issue-2759070320)